src/formElementPlugins/SummaryViewWrapper.tsx

```diff
@@ -1,12 +1,58 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { ErrorBoundary, pluginProvider } from '.'
+import { Grid, Icon } from 'semantic-ui-react'
+import { SummaryViewWrapperProps, PluginComponents, ValidationState } from './types'
 import { Form, Header } from 'semantic-ui-react'
-import { SummaryViewWrapperProps, PluginComponents } from './types'
 import { TemplateElementCategory } from '../utils/generated/graphql'
+import { defaultValidate } from './defaultValidate'
+import { EvaluatorParameters } from '../utils/types'
+import { IQueryNode } from '@openmsupply/expression-evaluator/lib/types'
 
 const SummaryViewWrapper: React.FC<SummaryViewWrapperProps> = (props) => {
-  const { element, response } = props
+  const { element, response, allResponses, isStrictValidation } = props
   const { parameters, category, code, pluginCode, isEditable, isRequired, isVisible } = element
+  const { validation: validationExpression, validationMessage } = parameters
+  const [validationState, setValidationState] = useState<ValidationState>({} as ValidationState)
+  const responses = { thisResponse: response?.text, ...allResponses }
+  const [pluginMethods, setPluginMethods] = useState({
+    validate: (
+      validationExpress: IQueryNode,
+      validationMessage: string,
+      evaluatorParameters: EvaluatorParameters
+    ): any =>
+      defaultValidate(validationExpression, validationMessage, {
+        objects: [responses],
+        APIfetch: fetch,
+      }),
+  })
```

Comment:
I think this could be separated in `initiateGenericValidation` function and here only calls that. Thinking this would improve readability of this method.

Reply:
I agree that this isn't that easy to make sense of (I didn't write it originally ;) ) but we already have `defaultValidate` function, so having a whole other function that just initiates this function doesn't seem to help with confusion for me.

Reply:
Alright, but it's still convoluted there. Could you at least separate the types definition from the `validate` function definition on the useState declaration? In other words, create a type for `ValidateProps` and only list the properties received and pass down to the `defaultValidate`?

Reply:
Okay, I've defined a type for ValidateFunction and ValidateObject, which helps a bit. Let me know if that's okay.